Adding new ppl profiles (#558)
usds/website: _people/0650_eunice-garcia.md

---
# Page template info (DO NOT EDIT)
layout: default
banner_file: banner--people-projects-lg.svg
banner_file_mobile: banner--people-projects-sm.svg
people_page: true

# Page link (Edit this)
permalink: people/eunice-garcia

# People Carousel and Page (Edit this)
name: Eunice Garcia
pronouns: She/Her
alumni: false
cop: Front Office
city_of_origin: Los Angeles, CA
previously_at: Office of Presidential Records
image_name: eunice-garcia.jpg
quote: "There is definitely a need for more women of color to speak tech and government at the same time. I believe in changing the status quo, and more importantly, empowering good people with the right tools especially in government."

---

**What’s your background?**

I originally came to DC from California six years ago to work at a national security non-profit organization. Then I moved into federal government with the Office of Presidential Correspondence (OPC) in the previous Administration, five years ago. I’ve been in government ever since, jumping around to different teams. USDS is my first civic tech role.

**What inspired you to join USDS?**

> I first heard about USDS when I was at OPC. They had reached out to us looking for user stories from Veterans about their interactions with VA. They were very much the cool kids in the room, wearing jeans and hoodies amidst the sea of suits. What struck me was how passionate they were about their work, and that spoke to me. I am mission driven and aim to work in spaces that deliver impact, even behind the scenes. It’s inspiring to see the work USDS has done, and continues to do, specifically in a very complicated environment in unprecedented times.

Reviewer: ```suggestion
I first heard about USDS when I was at OPC. They had reached out to us looking for user stories from Veterans about their interactions with the Department of Veterans Affairs (VA). They were very much the cool kids in the room, wearing jeans and hoodies amidst the sea of suits. What struck me was how passionate they were about their work, and that spoke to me. I am mission-driven and aim to work in spaces that deliver impact, even behind the scenes. It’s inspiring to see the work USDS has done, and continues to do, specifically in a very complicated environment in unprecedented times.
```

**What has been your biggest challenge?**

> Owning my game, (a.k.a. imposter syndrome). I think I share similar sentiments, especially with many women on this one. There are days where my confidence lacks and yet you are depended on making crucial decisions for the well-being of the team.

Reviewer: ```suggestion
Owning my game, (a.k.a. imposter syndrome). I think I share similar sentiments, especially with many women on this one. There are days where my confidence lacks, and yet I'm depended on to make crucial decisions for the well-being of the team.
```

> It’s not always easy to say, “I’ve totally got this,” or “everything will be fine.” There is a lot more at play — hurdles, roadblocks, red tape, etc. and learning how to navigate that maze can be hard. I’ve learned to rely on bridges and encouraging voices in order to move forward.

Reviewer: ```suggestion
It’s not always easy to say, “I’ve totally got this,” or “everything will be fine.” There is a lot more at play — hurdles, roadblocks, red tape, etc. — and learning how to navigate that maze can be hard. I’ve learned to rely on bridges and encouraging voices in order to move forward.
```

**How does your work make an impact?**

> My impact is team focused. The role of being an Ops Manager is unique in the sense that a jack-of-all-trades. I see my role as an enabler, a guardian, a gatekeeper of the team.

Reviewer: ```suggestion
My impact is team-focused. My role as an operations manager is unique in the sense that I'm a jack-of-all-trades. I see myself as an enabler, a guardian, a gatekeeper of the team.
```

> My job is to create a healthy environment, eliminate unnecessary fires, support, guide, and facilitate. Whether it’s ensuring we have a functional work space, the right tech equipment, helping newbies find their way, or wrangling ethics clearances for an event, there are a lot of moving pieces that demand quality service on my end to ensure the team’s ability to delivery working product.

Reviewer: ```suggestion
My job is to create a healthy environment, put out fires, support, guide, and facilitate. Whether it’s ensuring we have a functional work space and the right tech equipment, helping newbies find their way, or wrangling ethics clearances for an event, there are a lot of moving pieces that demand quality service on my end to ensure the team’s ability to deliver working products.
```

**What do you want to do after USDS?**

> Continue to work in the civic tech space. There is definitely a need for more women of color to speak tech and government at the same time. I believe in changing the status quo, and more importantly, empowering good people with the right tools especially in government, which could bring everlasting change.

Reviewer: ```suggestion
I want to continue to work in the civic tech space. There is definitely a need for more women of color to speak tech and government at the same time. I believe in changing the status quo, and more importantly, empowering good people with the right tools, especially in government, which could bring everlasting change.
```
